src/Client/Client.php

```diff
@@ -100,41 +88,20 @@ public function request(string $url, string $method = 'GET', string $body = null
             ->sendRequest($request);
     }
 
-    /**
-     * @param string $url
-     * @param array $parameters
-     * @param string[] $headers
-     * @return ResponseInterface
-     * @throws HttpException
```

Comment:
`@throws` is niet af te leiden van de type hints. Dus die mag blijven staan. (geld voor alle methods)